src/OpenRealEstate.Core/ISaleDetails.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OpenRealEstate.Core
+{
+    public interface ISaleDetails
+    {
+        string YearBuilt { get; set; }
+
+        string YearLastRenovated { get; set; }

```

Comment:
What are some values here? could this be an `int` instead?

Also possible to make it a `DateOnly` but I think that's an overkill.

Reply:
they are a numeric year value, could use int but i have an aversion to that, but your call

Reply:
`int` please. I don't want to see a value of `hello`. etc

Reply:
updated